### PR TITLE
Travis update

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,10 +15,6 @@ cache:
     - $HOME/.composer/cache
     - $HOME/.cache/pip
 
-env:
-  global:
-    - SYMFONY_DEPRECATIONS_HELPER=weak
```
Comment (OskarStark) [inline .travis.yml]: why removing this line?

**Comment by OskarStark** [comment]
ping @Soullivaneuh

File: .travis.yml (hunk 1 above)
Comment (Soullivaneuh) [inline .travis.yml]: This is related to SF 2.8 and 3.0.

But, I think we should keep it and prepare for 3.1

File: composer.json [hunk 2]
```diff
@@ -23,12 +23,14 @@
         "twig/twig": "~1.12"
     },
     "require-dev": {
-        "sonata-project/user-bundle": "~2.2",
```
Comment (Soullivaneuh) [inline composer.json]: * Not related to Travis.
* :-1: Anyway, it's a dev require for test only. This should be kept.

**Comment by Soullivaneuh** [comment]
Why closing this one?

Just some change have to be done.

**Pull request closed without merging**